sync-orders-to-bigQuery/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "moltin-google-bigquery",
```

Comment:
Can we rename this directory to `sync-orders-to-big-query`

Reply:
Removed this field altogether following example on https://github.com/moltin/integration-examples/blob/master/sync-catalog-to-algolia/package.json